src/main/java/com/exasol/releasedroid/progress/Progress.java

```diff
@@ -1,62 +1,108 @@
-package com.exasol.releasedroid.adapter.github.progress;
+package com.exasol.releasedroid.progress;
 
 import static com.exasol.releasedroid.formatting.Colorizer.*;
 
 import java.time.*;
 import java.time.format.DateTimeFormatter;
-import java.util.Date;
-import java.util.Optional;
 
-public class ProgressFormatter {
+// [impl->dsn~progress-display~1]
+public class Progress {
+
+    /**
+     * Return a silent progress not printing anything when asked
+     */
+    public static final Progress SILENT = new SilentProgress();
 
     public static Builder builder() {
         return new Builder();
     }
 
+    private static final boolean ECLIPSE_CONSOLE = false;
```

Comment:
I guess this is only used for testing? If you want to keep it, please add a comment.

Reply:
👍